src/js/components/returns/outbound/AddItemsPage.jsx

```diff
@@ -323,7 +324,7 @@ class AddItemsPage extends Component {
 
   dataFetched = false;
 
-  saveAndTransition(callback) {
+  saveAndTransition(callback, status) {
```

Comment:
Having a default value could be clearer: `saveAndTransition(callback, status=null) {`

Though I haven't worked with react much so maybe it's well enough understood behaviour that it'll be undefined if not provided